tensorleap/helm-charts
# add go installation as library

- Add make file command (test, fmt, lint)
- Initilaize go.mod and dependencies
- Move all installation code
- Get onInstalled hook function on NewInstallCmd for login
- Create the tools cmd with dedeceted function for reuse
- Check that the tag start with tensorleap- to avoid using worng tag (go release tag)
- Add Go CI workflow
